t/decode-base64.t

```diff
@@ -9,7 +9,7 @@ use Cwd qw(cwd);
 
 repeat_each(2);
 
-plan tests => repeat_each() * (blocks() * 5);
+plan tests => repeat_each() * ((blocks() * 5) - 2);
```

Comment:
Seems like we can save the parens around the `*` expression since it's obvious that `*` takes precedence over `-`.